solutions/ai-chatgpt/components/Chat.tsx

```diff
@@ -58,42 +58,68 @@ export function Chat() {
     }
   }, [cookie, setCookie])
 
-  // send message to API /api/chat endpoint
-  const sendMessage = async (message: string) => {
-    setLoading(true)
-    const newMessages = [
-      ...messages,
-      { message: message, who: 'user' } as Message,
-    ]
-    setMessages(newMessages)
-    const last10messages = newMessages.slice(-10)
-
-    const response = await fetch('/api/chat', {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        messages: last10messages,
-        user: cookie[COOKIE_NAME],
-      }),
-    })
-    const data = await response.json()
-
-    // strip out white spaces from the bot message
-    const botNewMessage = data.text.trim()
+
+// send message to API /api/chat endpoint
+const sendMessage = async (message: string) => {
+  setLoading(true);
+  const newMessages = [
+    ...messages,
+    { role: "user", content: message } as ChatGPTMessage,
+  ];
+  setMessages(newMessages);
+  const last10messages = newMessages.slice(-10); // remember last 10 messages
+
+  const response = await fetch("/api/chat", {
+    method: "POST",
+    headers: {
```

Comment:
The indentation here is wrong, the function is still part of `Chat`. Can you run prettier over the files here 🙏

Reply:
Good call. I really don't know why it broke prettier. I had to reinstall it and it worked.